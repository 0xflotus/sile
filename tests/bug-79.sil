\begin[papersize=a5]{document}
\font[size=10pt, family=Linux Libertine O, language=cs]
\set[parameter=linebreak.emergencyStretch, value=0.5cm]
\begin[first-content-frame=frame]{pagetemplate}
\frame[id=frame, left=2cm, width=6cm, top=2cm, bottom=10cm]
\end{pagetemplate}%
zana Bourová, zápolila ve čtyřech závodech, nejlépe se umístila na Benecku na 4. místě. Magdaléna Chrástová skončila na místě sedmnáctém, Barbora Šulcová byla hned za
\end{document}