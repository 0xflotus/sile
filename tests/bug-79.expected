<tests/bug-79.sil>
Set paper size 	419.5275636	595.275597
Begin page
Mx 	76.69291
My 	63.67291
Set font 	Linux Libertine O;10;200;normal;normal;;LTR
T	91 66 79 66	(zana)
Mx 	101.28510
T	35 80 86 83 80 87 161	(Bourová)
Mx 	135.66510
T	13	(,)
Mx 	143.65728
T	91 161 81 80 77 74 77 66	(zápolila)
Mx 	181.11947
T	87 70	(ve)
Mx 	196.27166
T	205 85 90 281 70 68 73	(čtyřech)
Mx 	56.69291
My 	77.67291
T	91 161 87 80 69 70 68 73	(závodech)
Mx 	94.76291
T	13	(,)
Mx 	104.05066
T	79 70 75 77 169 81 70	(nejlépe)
Mx 	140.58841
T	84 70	(se)
Mx 	156.04616
T	86 78 173 84 85 74 77 66	(umístila)
Mx 	196.03391
T	79 66	(na)
Mx 	213.11166
T	35 70	(Be)
Mx 	223.46166
T	14	(-)
Mx 	56.69291
My 	91.67291
T	79 70 68 76 86	(necku)
Mx 	84.91666
T	79 66	(na)
Mx 	98.46041
T	21	(4)
Mx 	103.11041
T	15	(.)
Mx 	108.86416
T	78 173 84 85 219	(místě)
Mx 	131.00416
T	15	(.)
Mx 	136.75791
T	46 66 72 69 66 77 169 79 66	(Magdaléna)
Mx 	185.00166
T	36 73 83 161 84 85 80 87 161	(Chrástová)
Mx 	56.69291
My 	105.67291
T	84 76 80 79 205 74 77 66	(skončila)
Mx 	94.97510
T	79 66	(na)
Mx 	109.56728
T	78 173 84 85 219	(místě)
Mx 	136.30947
T	84 70 69 78 79 161 68 85 169 78	(sedmnáctém)
Mx 	187.50947
T	13	(,)
Mx 	194.31166
T	35 66 83 67 80 83 66	(Barbora)
Mx 	56.69291
My 	119.67291
T	288 86 77 68 80 87 161	(Šulcová)
Mx 	91.29327
T	67 90 77 66	(byla)
Mx 	111.59364
T	73 79 70 69	(hned)
Mx 	135.00400
T	91 66	(za)
[1] Mx 	207.41759
My 	553.73265
Set font 	Gentium;10;200;normal;normal;;LTR
T	17	(1)
End page
Finish

